static/sass/shared.scss

```diff
@@ -77,3 +77,7 @@ button:not(:disabled):active {
     display: initial;
   }
 }
+
+.no-web-share {
```

Comment:
want to combine this with the `.no-push-notifications` selector?

Reply:
Done.